{"changed": false, "cmd": "./manage.py makemigrations", "failed": true, "msg": "\n:stderr: Traceback (most recent call last):\n  File \"./manage.py\", line 22, in <module>\n    execute_from_command_line(sys.argv)\n  File \"/vagrant/projectenv/local/lib/python2.7/site-packages/django/core/management/__init__.py\", line 367, in execute_from_command_line\n    utility.execute()\n  File \"/vagrant/projectenv/local/lib/python2.7/site-packages/django/core/management/__init__.py\", line 341, in execute\n    django.setup()\n  File \"/vagrant/projectenv/local/lib/python2.7/site-packages/django/__init__.py\", line 22, in setup\n    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)\n  File \"/vagrant/projectenv/local/lib/python2.7/site-packages/django/utils/log.py\", line 75, in configure_logging\n    logging_config_func(logging_settings)\n  File \"/usr/lib/python2.7/logging/config.py\", line 794, in dictConfig\n    dictConfigClass(config).configure()\n  File \"/usr/lib/python2.7/logging/config.py\", line 576, in configure\n    '%r: %s' % (name, e))\nValueError: Unable to configure handler 'file': [Errno 2] No such file or directory: '/vagrant/transaction_simulator/logs/debug.log'\n", "path": "/vagrant/projectenv/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games", "state": "absent", "syspath": ["/tmp/ansible_hclMCn", "/tmp/ansible_hclMCn/ansible_modlib.zip", "/tmp/ansible_hclMCn/ansible_modlib.zip", "/usr/lib/python2.7", "/usr/lib/python2.7/plat-x86_64-linux-gnu", "/usr/lib/python2.7/lib-tk", "/usr/lib/python2.7/lib-old", "/usr/lib/python2.7/lib-dynload", "/usr/local/lib/python2.7/dist-packages", "/usr/lib/python2.7/dist-packages"]}
